### PR TITLE
Pop Up windpws

### PR DESCRIPTION
**Pull request by Arturgouveia1970**
we will implement in this PR the popup windows for both the mobile and desktop versions.
so far I have created and styled, both versions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -105,92 +105,14 @@ <h3>Multi-Post Stories</h3>
...
-            <li><img class="langicon" src="images/html2.png" alt="html logo"></li>
-            <li><img class="langicon" src="images/bootstrap2.png" alt="bootstrap logo"></li>
-            <li><img class="langicon" src="images/ruby2.png" alt="ruby logo"></li>
-          </ul>
-        </div>
-        <button type="button" class="btn">See Project</button>
+        <button type="button" class="btn-2">See Project</button>
       </section>
+      
+      <section class="projects" id="mobile-proj"></section>
```
Comment (OLIPLICHE) [inline index.html:111]: - [ ] Please the `projects section` is created dynamically with the information from the projects array. Remember the [Project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md): All of the HTML in that section is created when the page loads.
Therefore, What you need to do is:
- Kindly change the existing code; the already created HTML code for the `projects section` list should be removed, and instead it should be added via JavaScript

File: index.html [hunk 2]
```diff
@@ -338,7 +263,59 @@ <h4>I'm always interested in hearing about new projects,
...
+            
+            the releax map lapora verita.
+        </p>
+        <div class="buttons">
+          <button type="button" class="see-live-btn"></button>
+          <button type="button" class="see-source-btn"></button>
+        </div>
+        </div>
+      </div>
+    </section>
```
Comment (OLIPLICHE) [inline index.html:318]: - [ ] Please Modal popup data is populated with the data from the projects array. We don't need to see the popup code inside HTML.
  Kindly remember your [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md): In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects...

- [ ] Kindly make sure the popup is responsive, it fits mobile and desktop sizes properly

File: mobile-menu.js [hunk 3]
```diff
@@ -13,4 +13,124 @@ mobileBtnExit.addEventListener('click', () => {
...
+          <h3 class="title-post">${proj.name}</h3>
+          <p>${proj.description}</p>
+          <ul class="icons">
+            <li><img class="langicon" src="images/html2.png" alt="html logo"></li>
+            <li><img class="langicon" src="images/bootstrap2.png" alt="bootstrap logo"></li>
+            <li><img class="langicon" src="images/ruby2.png" alt="ruby logo"></li>
+          </ul>
+          <button type="button" class="btn-2">See Project</button>
+`).join(''); */
+
```
Comment (OLIPLICHE) [inline mobile-menu.js:125]: - [ ] Please remove (delete) commented code to make your code Dry, clean, and more professional

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Linter result is green ✔️
 Commit messages are meaningful✔️
 The pull request have a descriptive title telling you what changes are inside ✔️
 The pull request have a short summary describing introduced changes in more detail ✔️
 node_modules folder is in the .gitignore ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brytebee — APPROVED** [review]
Hi Team👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**